### PR TITLE
Updated logo dimensions and made it fluid #1403

### PR DESCRIPTION
**Pull request by kkoryaka**
Remove hardcoded width. made logo size compatible, responsive and fluid

### CODE REVIEW

**Review by carterm — CHANGES_REQUESTED** [review]
Need to specify a width or an aspect ratio to prevent content shift.  Load the page with caching off and 3G and you will see the header jump without a width.

**Review by jon-grant-state-ca-gov — APPROVED** [review]
Looks good!

**Comment by carterm** [comment]
Screen Shot 2023-05-19 at 3 46 16 PM
Content shift

**Comment by kkoryaka** [comment]
> Need to specify a width or an aspect ratio to prevent content shift. Load the page with caching off and 3G and you will see the header jump without a width.

@carterm I did't touch the aspect ratio in the side-header.njk file, it should be there for this site.

File: src/scss/cagov/site-header.scss [hunk 1]
```diff
@@ -32,14 +31,12 @@
     }
 
     img {
-      width: 100%;
```
Comment (carterm) [inline src/scss/cagov/site-header.scss:35]: Need to replace with an aspect ratio if removing width

**Comment by jon-grant-state-ca-gov** [comment]
Latest commit fixed content shift on my machine

**Review by carterm — APPROVED** [review]
No more content shift

**Pull request merged**